medusa/scene_exceptions.py

```diff
@@ -15,14 +15,14 @@
 from six import iteritems
 from . import app, db, helpers
 from .indexers.indexer_config import INDEXER_TVDBV2
-from .session.core import MedusaSession
+from .session.core import MedusaSafeSession
 
 logger = logging.getLogger(__name__)
 
 exceptions_cache = defaultdict(lambda: defaultdict(set))
 exceptionLock = threading.Lock()
 
-xem_session = MedusaSession()
+xem_session = MedusaSafeSession()
```

Comment:
@p0psicles this `xem_session` is imported in scene_numbering.py so I prefer to kept the name